README.md

```diff
@@ -142,7 +145,81 @@ export const rLogin = new RLogin({
 })
 ```
 
-> We usually put this all together in a single file called `rlogin.ts` and export a single instance of `RLogin`. This ensures a single DOM element is creted.
+#### Method 2. 
+Provide an array of supported network parameters to the rLogin constructor within `ethereumChains` parameter. This will allow you to add the selected network to Metamask (if it has not been previously added) and switch to the network if the wallet was initially connected to an unsupported network.
```

Comment:
Has this been tested with the other providers that are listed below, specifically WC and the hardware wallets? If not, let me know which ones we need to test manually.